cmd/subnetcmd/deploy.go

```diff
@@ -221,19 +228,28 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 		}
 		return nil
 
-	case models.Fuji: // just make the switch pass
-		if keyName == "" {
-			keyName, err = captureKeyName()
+	case models.Fuji:
```

Comment:
I already reused part of the code in ChooseKeyOrLedger. Also there is shared code on captureKeyName. I would prefer to keep the other main logic duplicated. Also because it is modifiying useLedger variable.

Reply:
But if you insist and tell me which part you would like to extract, I am open to that